vir/defs/high/operations_internal/expression.rs

```diff
@@ -72,6 +103,16 @@ impl Expression {
             _ => false,
         }
     }
+    pub fn erase_lifetime(&mut self) {
```

Comment:
A canonical way of doing this would be to have a folding visitor that replaces all types. Similarly to this: https://github.com/viperproject/prusti-dev/blob/6a9291b92f1d742e36715a3fde15786abd950ea8/vir/defs/high/operations_internal/expression.rs#L245-L260

Reply:
agree, will do